scalalib/src/mill/javalib/android/AndroidModule.scala

```diff
@@ -706,4 +708,248 @@ trait AndroidModule extends JavaModule {
     PathRef(signedApk)
   }
 
+  /** Whether we are building an application (APK) or a library (AAR) */
+  def isApplication: T[Boolean] = Task {
```

Comment:
why do we need this? we have AndroidModule and AndroidAppModule which should separate the two